server/serverRun.py

```diff
@@ -199,11 +216,12 @@ def items():
                 'id': item.id
             }
             for item in user_items
-        ]
-    return json.dumps(user_items)
+    ]
+    return json.dumps(user_items), 200, JSON_CT
 
 
 @app.route('/item', methods=['POST'])
+@logged_in()
 def create_item():
```

Comment:
Do we still need this redirect on the server side? Vue will redirect the page back to /login  We just need this endpoint to invalidate the cookie so that they need to /login again to regain access to the API.

Reply:
Hmm, I'm noticing that perhaps my comments aren't in the right place in the code. 😂 I was trying to review from a mobile app, once I get to work I'll see if I can fix this so at least the code snippet shows correctly.

Reply:
🤔 I was so confused lmao